test/message/error_chainErrors.js

```diff
@@ -0,0 +1,10 @@
+// Flags: --expose-internals
+'use strict';
+
+require('../common');
+const { chainErrors } = require('internal/errors');
```

Comment:
```suggestion
const { aggregateTwoErrors } = require('internal/errors');
```
Should this file be named `test/message/error_aggregateTwoErrors.js` instead?